### PR TITLE
Add css variable fallback for ie

### LINKED ISSUE #114: Feature - Browser support for css vars
**Is your feature request related to a problem? Please describe.**
CSS Vars is not working in IE & Edge

**Describe the solution you'd like**
Polyfill for css vars

https://github.com/jhildenbiddle/css-vars-ponyfill

### PR DESCRIPTION
**Pull request by awinny**
**Describe pull-request**</br>
- Added fallback colors for css variables
- Added global true as default setting

**Solving issue**</br>
Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)
Fixes: #114

### CODE REVIEW

**Comment by awinny** [comment]
Notice that #119 should be merged before this is merged. Due to this branch containing the same feature as in that pull request.

File: src/themes/scania/c-theme.scss [hunk 1]
```diff
@@ -148,6 +150,12 @@ $gray-colors: (
   'gray-09': $gray-900
 );
 
+$element-colors: (
+  'headings-color': var(--headings-color),
```
Comment (helloimela) [inline src/themes/scania/c-theme.scss]: --headings-color renders #53565a
It should be $theme-01 : #041e42

Comment (helloimela) [reply src/themes/scania/c-theme.scss]: This problem still appear in IE

**Comment by helloimela** [comment]
please check the headings color in IE, and footer dropup toggle color is blue in IE

**Review by helloimela — APPROVED** [review]

**Pull request merged**